### ISSUE TITLE
[Feature request] Support arbitary long sequences.

### ISSUE THREAD

**Issue by bratao** [BrikerMan/Kashgari#95]
Hello @BrikerMan ,
Again thank you for this incredible work.

As discussed in another issue, for some tasks we have an arbitrary long sequence size. And right now Kashgari requires to set a fixed size beforehand. 
This size cannot be too big, because of vanishing gradients problem( for RNN) and the non convergence of LSTM.

https://machinelearningmastery.com/handle-long-sequences-long-short-term-memory-recurrent-neural-networks/ provides some nice solutions for this problem. 

My observation about the viability to Kashgari, I see those two options as more promising:

**Truncated Backpropagation Through Time**
Looks like the implementation is not too hard, but no clear support from Tensorflow/keras.

**Stateful LSTM**
This looks like the better option for Kashgari right now.
Tensorflow and Keras supports stateful LSTM. But it will require some house keeping such as:
-  Split the input sequence into multiple fixed-length sub-sequences.
- No shuffling between the splitted sequences
- Manually reset the LSTM state between instances.

Do you think that this feature would be possible to support in Kashgari?

Thank you!

**Comment by BrikerMan** [comment]
@bratao We have just released the tf.keras version and [new website](https://kashgari.bmio.net/). Maybe you could help us with the documents. Trying to make this part more simple and clean.

```
Kashgari is a simple and powerful NLP Transfer learning framework, build a state-of-art model in 5 minutes for named entity recognition (NER), part-of-speech tagging (PoS) and text classification tasks.
```
